ui/src/external/monaco.flux.autogrow.ts

```diff
@@ -0,0 +1,41 @@
+const MIN_HEIGHT = 100
+
+export const registerAutogrow = editor => {
+  let prevHeight = 0
+
+  const updateEditorHeight = () => {
+    const editorElement = editor.getDomNode()
+
+    if (!editorElement) {
+      return
+    }
+
+    const lineHeight = editor.getOption(
+      window.monaco.editor.EditorOption.lineHeight
+    )
+    const lineCount = (editor.getModel() || {}).getLineCount() || 1
+    const height = editor.getTopForLineNumber(lineCount + 1) + lineHeight
+
+    if (prevHeight !== Math.max(MIN_HEIGHT, height)) {
+      prevHeight = Math.max(MIN_HEIGHT, height)
+      editorElement.style.height = `${height}px`
```

Comment:
should this be setting the height to the reassigned `prevHeight`? It doesn't seem like we're using `prevHeight` anywhere else

Reply:
you're probably right. lemme double check